spec/models/authenticator/ldap_spec.rb

```diff
@@ -163,6 +164,19 @@ def authenticate
     let(:username) { 'alice' }
     let(:password) { 'secret' }
 
+    context "when using LDAP" do
+      let(:config) { super().merge(:ldap_role => true) }
+
+      it "encrypts password for queuing" do
+        called_encrypt_ldap_password = false
+        allow(MiqTask).to receive(:create).and_return(double(:id => :return_value))
+        allow(MiqQueue).to receive(:put) { :return_value }
+        allow(subject).to receive(:encrypt_ldap_password) { called_encrypt_ldap_password = true }
+        authenticate
+        expect(called_encrypt_ldap_password).to be_truthy
```

Comment:
Hmm I'm not sure what's going on here. This could easily be a false positive as without any of the preceding lines `called_encrypt_ldap_password` could be true.